addons/l10n_cl/models/account_journal.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class AccountJournal(models.Model):
+
+    _inherit = "account.journal"
+
+    l10n_cl_sequence_ids = fields.Many2many('ir.sequence', relation='l10n_cl_journal_sequence_rel',
+                                            column1='journal_id', column2='sequence_id', string='Sequences',
+                                            domain="[('l10n_latam_document_type_id', '!=', False)]")
+
+    def create_document_sequences(self):
+        self.ensure_one()
+        if self.company_id.country_id != self.env.ref('base.cl'):
+            return True
+
+        if (self.env['ir.sequence'].search([('l10n_latam_document_type_id', '!=', False)]) and
+                not self.env.context.get('manual_creation', False)):
```

Comment:
Now way to avoid this context?  By putting the creation itself in a separate method e.g.?